test/azul_test_case.py

```diff
@@ -82,8 +87,15 @@ def dummy_get_credentials(self):
         botocore.session.Session.get_credentials = dummy_get_credentials
         boto3.session.Session.get_credentials = dummy_get_credentials
 
+        # Ensure that mock leakages fail by targeting another region
+        # Moto will verify that region is valid.
+        # We are pretty much guaranteed not to have access to this region.
+        cls._aws_region_mock = patch.dict(os.environ, AWS_DEFAULT_REGION='us-gov-west-1')
```

Comment:
Any other gov region we could use?

Reply:
I don't believe so: https://github.com/DataBiosphere/azul/pull/1710#discussion_r410097292

If it's that important I could try to mock moto's internals so that a made-region is mapped to a valid backend.

Reply:
Actually that almost certainly won't work. Moto retrieves the list of available regions from boto at import time so it's too late for `setUpClass` to patch anything. Furthermore the separate moto services (e.g. dynamodb, sqs) have their own regions lists, so to overwrite boto's list we'd need to patch each each individually, which seems very brittle and tedious. My current belief is that our best option is to select one the regions listed above.